### ISSUE TITLE
[InvalidArgumentException] $absolutePath must be relative to "files"

### ISSUE THREAD

**Issue by dasboeh** [Rello/audioplayer#316]
### Expected behavior
Command "sudo -u www-data php occ -v audioplayer:scan --all" in /var/www/nextcloud scans all files on Nextclud instance.

### Actual behavior
The above command throws the followinng exception:
`[InvalidArgumentException]                 
  $absolutePath must be relative to "files" 

Exception trace:
 () at /var/www/nextcloud/lib/private/Files/View.php:2111
 OC\Files\View->getPathRelativeToFiles() at /var/www/nextcloud/lib/private/Files/View.php:1930
 OC\Files\View->lockPath() at /var/www/nextcloud/lib/private/Files/View.php:2034
 OC\Files\View->lockFile() at /var/www/nextcloud/lib/private/Files/View.php:1134
 OC\Files\View->basicOperation() at /var/www/nextcloud/lib/private/Files/View.php:267
 OC\Files\View->mkdir() at /var/www/nextcloud/lib/private/Files/Node/Folder.php:160
 OC\Files\Node\Folder->newFolder() at /var/www/nextcloud/lib/private/Files/Node/Root.php:380
 OC\Files\Node\Root->getUserFolder() at n/a:n/a
 call_user_func_array() at /var/www/nextcloud/lib/private/Files/Node/LazyRoot.php:64
 OC\Files\Node\LazyRoot->__call() at /var/www/nextcloud/lib/private/Files/Node/LazyRoot.php:281
 OC\Files\Node\LazyRoot->getUserFolder() at /var/www/nextcloud/apps/audioplayer/lib/Controller/ScannerController.php:434
 OCA\audioplayer\Controller\ScannerController->getAudioObjects() at /var/www/nextcloud/apps/audioplayer/lib/Controller/ScannerController.php:145
 OCA\audioplayer\Controller\ScannerController->scanForAudios() at /var/www/nextcloud/apps/audioplayer/lib/Command/Scan.php:77
 OCA\audioplayer\Command\Scan->execute() at /var/www/nextcloud/3rdparty/symfony/console/Command/Command.php:264
 Symfony\Component\Console\Command\Command->run() at /var/www/nextcloud/3rdparty/symfony/console/Application.php:874
 Symfony\Component\Console\Application->doRunCommand() at /var/www/nextcloud/3rdparty/symfony/console/Application.php:228
 Symfony\Component\Console\Application->doRun() at /var/www/nextcloud/3rdparty/symfony/console/Application.php:130
 Symfony\Component\Console\Application->run() at /var/www/nextcloud/lib/private/Console/Application.php:173
 OC\Console\Application->run() at /var/www/nextcloud/console.php:90
 require_once() at /var/www/nextcloud/occ:11
`

### Steps to reproduce the behavior
1. cd /var/www/nextcloud
2. sudo -u www-data php occ audioplayer:reset --all
3. sudo -u www-data php occ audioplayer:scan --all

### Additional information (optional)
On Webinterface: Scan starts and after about 10 to 20 files just stopps. Music library is 20.300 files.

### Server configuration
**Operating system:**
Ubuntu 18.04

**Web server:**
Nginx

**Database:**
MariaDB

**PHP version:**
7.2

**Nextcloud/ownCloud version:** (see `/status.php`)
13.0.4

**Audio Player version:** (see `/settings/apps`)
2.3.0

**Where did you install Audio Player from?** `NC App Store/oC Marketplace/GitHub`
NC App Store

**Are you using external storage? If yes, which one:** `local/smb/sftp/...`
no, but NC Data is on /data/nextcloud, which is a mountpoint for /dev/sda

**Are you using encryption?** yes/no
no

**Have you activated other multimedia apps?** `Music/Folder Player/...`
Video Player

**Audio Player media folder:** (see Settings) `root/Music/...`
'root/Musik'

**Comment by Rello** [comment]
Hello,
thank  you for reporting any issues.
can you please try to execute the off with the `--debug `parameter at the end?
I would like to know if it fails at the same file every time, of if it processes e.g. 10 by 10

please let me know if we can narrow down this issue...

**Comment by dasboeh** [comment]
Hello Rello.

I started the scan in debug mode and it ran way farther than I thought. It stopped a few thousand files later because a file was locked. I emptied the oc_lock_files table in mysql as described here: https://help.nextcloud.com/t/file-is-locked-how-to-unlock/1883 and voila: the next scan finished without exception.
So the bug was on my side. Thank you for pointing the --debug option out to me.

Regards
Stefan
